packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -216,7 +219,11 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
     private doHandleMarkerFilterChangedSignal(annotationMarkers: string[]) {
```

Comment:
Now the Markers filter toolbar button contents is dependent on the current trace context. But when the filter is applied, it affects the annotations (in the time graph) of every output of every trace. So at that point a trace's visible annotations does not match its own Markers filter state.